itest/lnd_estimate_route_fee_test.go

```diff
@@ -357,12 +353,6 @@ func testEstimateRouteFee(ht *lntest.HarnessTest) {
 			break
 		}
 	}
-
-	mts.ht.CloseChannelAssertPending(mts.bob, channelPointBobPaula, false)
```

Comment:
good q - my approach is that we should test a single behavior in one test, and the force close behaviors should be covered in `lnd_multi-hop_force_close_test.go` and `lnd_channel_force_close_test.go`.

Reply:
I'd argue that not starting from the same clean state can alter test outcomes in the extreme case, but as a plus it may end up finding more flakes. We still probably get more readable logs if we clean things up properly.

Reply:
def - we now no longer uses standby nodes, so we will always start with a clean state!